### PR TITLE
Add an end point to set a boleto as paid

### PR DESCRIPTION
**Pull request by bvicenzo**
This endpoint allow us to mark a boleto payment as paid. And continue our payment flow...

### CODE REVIEW

File: spec/sales_spec.rb [hunk 1]
```diff
@@ -380,4 +380,85 @@
...
+            put "/v2/sales/2014111703/conciliate"
+
+            response = JSON.parse(last_response.body)
+
+            expect(last_response).to be_ok
+            expect(response).to eq([{'Code' => 114, 'Message' => 'Error'}])
+          end
+        end
+
+        context 'with conciliate toggle enabled' do
```
Comment (dkonishi) [inline spec/sales_spec.rb]: s/enabled/disabled/ ?

File: lib/fake_braspag/sales.rb [hunk 2]
```diff
@@ -40,6 +40,16 @@ class Sales < Sinatra::Base
       end
     end
 
+    put '/:PaymentId/conciliate' do
+      order = Order.find(params[:PaymentId])
+      if order.present? && order.PaymentMethod == 'Boleto' && ResponseToggler.enabled?('conciliate')
```
Comment (dkonishi) [inline lib/fake_braspag/sales.rb]: Maybe we can move `ResponseToggler.enabled?('conciliate')` to `if` beginning to shortcut this validation when  `ResponseToggler.enabled` is false. wdyt?

File: spec/sales_spec.rb (hunk 1 above)
Comment (bvicenzo) [inline spec/sales_spec.rb]: Yep! Thanks so much @dkonishi :smile:

File: lib/fake_braspag/sales.rb (hunk 2 above)
Comment (dkonishi) [inline lib/fake_braspag/sales.rb]: And if a 404 error will not be handled, we could even move the ` Order.find(params[:PaymentId])` to `if` block :smile:

Comment (dkonishi) [inline lib/fake_braspag/sales.rb]: And if a 404 error will not be handled, we could even move the ` Order.find(params[:PaymentId])` to `if` block
```ruby
if ResponseToggler.enabled?('conciliate') && order = Order.find(params[:PaymentId]) && order.PaymentMethod == 'Boleto'
  ...
end
```

File: spec/sales_spec.rb (hunk 1 above)
Comment (bvicenzo) [inline spec/sales_spec.rb]: @dkonishi fixed here: https://github.com/Baby-com-br/fake-braspag/commit/03780ac7b28c557f2f9f2a50b8fa9ebf9d8d988c

File: lib/fake_braspag/sales.rb (hunk 2 above)
Comment (bvicenzo) [inline lib/fake_braspag/sales.rb]: I don't know why, but, this way I got this errror:
Undefined method ```PaymentMethod``` for nil class... :disappointed:

**Comment by antoniofilho** [comment]
:shipit:

**Comment by pmatiello** [comment]
:sheep:

**Pull request merged**